examples/dbinit.py

```diff
@@ -0,0 +1,30 @@
+"""
+Register nodes with HaaS.
+
+This is intended to be used as a template for either creating a mock HaaS setup
+for development or to be modified to register real-life nodes that follow a
+particular pattern.
+"""
+
+from subprocess import check_call
+
+N_NODES=10
+
+ipmiUser = "ADMIN_USER"
```

Comment:
Variable names should be `lower_case_with_underscores`; this is standard python coding style. Please fix.